main.go

```diff
@@ -231,7 +228,11 @@ func filterData(pump pumps.Pump, keys []interface{}) []interface{} {
 		if filters.ShouldFilter(decoded) {
 			continue
 		}
-		filteredKeys[newLenght] = key
+		if !SystemConfig.RecordKeys {
```

Comment:
@letzya, is it ok to change the default behavior? What if instead of using RecordKeys, we use "ObfuscateKeys" so if the client wants to obfuscate, they can activate it.

Reply:
What is your reasoning for this change assuming it's the same just the other way around? (I was debating myself with that :) )